src/components/widgets/EditableList.vue

```diff
@@ -55,103 +56,107 @@ function commitEdit(index: number) {
 </script>
 
 <template>
-  <LabelCollapsable :label="entityNamePlural">
-    <div
-      v-for="(item, itemIndex) in items"
-      :key="itemIndex"
-      class="card p-3 pb-1 mb-3 rounded-3 osim-editable-list-card"
-      :class="{
-        'bg-light-light-orange': modifiedItemIndexes.includes(itemIndex),
-        'bg-light-light-gray': !modifiedItemIndexes.includes(itemIndex),
-        'bg-light-green': !item.uuid,
-      }"
-    >
-      <div>
-        <div v-if="item.uuid" class="osim-list-edit">
-          <slot
-            v-if="isBeingEdited(itemIndex)"
-            name="edit-form"
-            v-bind="{ item, items, itemIndex, ...useModalForItem(item.uuid) }"
-          />
-          <slot
-            v-else
-            name="default"
-            v-bind="{ item, items, itemIndex, ...useModalForItem(item.uuid) }"
-          />
-
-          <div class="buttons">
-            <button
-              v-if="indexBeingEdited !== itemIndex"
-              type="button"
-              class="btn"
-              @click="setEdit(itemIndex)"
-            >
-              <i class="bi bi-pencil"><span class="visually-hidden">Edit {{ entityName }}</span></i>
-            </button>
-            <button
-              v-if="indexBeingEdited !== itemIndex"
-              type="button"
-              class="btn"
-              @click="useModalForItem(item.uuid).openModal"
-            >
-              <i class="bi bi-trash">
-                <span class="visually-hidden">Delete {{ entityName }}</span></i>
-            </button>
-            <button
-              v-if="indexBeingEdited === itemIndex"
-              type="button"
-              class="btn"
-              @click="commitEdit(itemIndex)"
-            >
-              <i class="bi bi-check">
-                <span class="visually-hidden">Confirm {{ entityName }} Edit </span>
-              </i>
-            </button>
-            <button
-              v-if="indexBeingEdited === itemIndex"
-              type="button"
-              class="btn"
-              @click="cancelEdit(itemIndex)"
-            >
-              <i class="bi bi-x">
-                <span class="visually-hidden">Cancel {{ entityName }} Edit </span>
-              </i>
-            </button>
+  <div class="my-3">
```

Comment:
I think we could just pass this class to LabelCollapsable and modify it to apply any classes passed in as attributes

Reply:
The thing is that the form on `EditableList` (buttons to add/save new list items) was intended to wrapped by the vertical margin, adding the class to `LabelCollapsable` would exclude the form. 

But I see your point on not using strict styles on reusable components, so I think will be best to add that class on the specific components on `FlawForm` (`IssueFieldAcknowledgments` & `IssueFieldReferences`)

Reply:
Ah, I just meant to suggest adding the functionality to the component to apply any classes passed to the component, but not to modify the component itself, so we could optionally apply any classes that we would want. I think I agree re the components in FlawForm, if I follow correctly.